js-old/src/ui/Certifications/certifications.js

```diff
@@ -19,7 +19,7 @@ import { connect } from 'react-redux';
 
 import { hashToImageUrl } from '~/redux/providers/imagesReducer';
 
-import defaultIcon from '../../../assets/images/certifications/unknown.svg';
+import defaultIcon from '~/../assets/images/certifications/unknown.svg';
```

Comment:
1st time I see this syntax, didn't know it existed! ~ represents the root right? I.e. where package.json is?

Reply:
In this case, it is the source root. It is a webpack alias - beats the ../../.. stuff that otherwise just litters the code.